### PR TITLE
add sascha1337 to cerem0ny

### PR DESCRIPTION
**Pull request by sascha1337**
f_1(r) = e(g^{aL(r)}, g^{bR(r)}) - e(g, g^{O(r) + m}) = e(g, g)^{aL(r) \times bR(r)} - e(g,g)^{O(r) + m}

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,8 +40,8 @@ $ snarkjs zkey contribute ./participations/$RECENT_ZKEY ./participations/$CONTRI
 - Verify your contribution
 
 ```bash
-$ wget https://hermez.s3-eu-west-1.amazonaws.com/powersOfTau28_hez_final_16.ptau
-$ snarkjs zkey verify ./build/withdraw.r1cs ./powersOfTau28_hez_final_16.ptau "./participations/$CONTRIBUTION_ID.zkey"
+wget https://hermez.s3-eu-west-1.amazonaws.com/powersOfTau28_hez_final_16.ptau
```
Comment (sascha1337) [inline README.md:43]: https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md014

File: README.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ sudo yarn global add snarkjs
 ````
 
 - Clone repository
+
```
Comment (sascha1337) [inline README.md:14]: https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md032

**Review by sascha1337 — COMMENTED** [review]
make markdownlint happy

**Comment by onewhiskeypls** [comment]
Hello, could you please redo your participation?  Unfortunately when there are open PR's, you'll have to fork off of that branch to preserve the circuit!

**Comment by sascha1337** [comment]
Sure, been pretty curious about it when i saw the open PRs, hoping that it
would work, so i pick the latest open pr as base and increment +1 kk

Thxx

**Pull request closed without merging**